examples/from-dockerfile/2.5/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubi8/ruby-25
```

Comment:
Shouldn't this file be in the `2.5` directory of this repo, not to repeateat another directory structure?

Reply:
It's an example and it belongs to the example directory, versions are there only due to tests

Reply:
Ok, but with Ruby f.e. we call this a `fixture` and it belongs right next to the tests.... I don't see why to add versioned examples when we have already versioned folders. It seems like enforced structure, because you wan't examples separated? But then again, shouldn't those be there for the ppl to use as a reference with the container? (Do they somehow iterfere with the resulting container?)

Reply:
I'm still against versioned examples, inside this repo, when we have already versioned folders for the respective Ruby versions. @hhorak @voxik WDYT? 

What I think: 

 1. The Ruby Dockerfile examples should not be versioned. Same way as the example application is not versioned.
 2. If it's aimed only on testing, it should be inside the Versioned (Ruby) folder, inside either `test` or `fixtures` folder (see above).

Note that if there's a worry to have an actual Dockerfile which is built for the specific version, we can use Docker ARGs, like this:

https://github.com/pvalena/tmt-all/blob/master/Dockerfile#L1

Build would look like:
```console
$ podman build --build-arg FROM='27' .
# or
$ podman build --build-arg FROM='ubi8/ruby-27' .
# or
$ podman build --build-arg FROM='ruby-27' .
```
Depending on what remains in Dockerfile.

Reply:
@pvalena, yes due to removing versioned examples I prepare PR  https://github.com/sclorg/s2i-ruby-container/pull/300 which enables SCL variables by default, and then we don't need versioned tests

Reply:
Great!